tests/pytests/test_themis_drivers.py

```diff
@@ -0,0 +1,397 @@
+                    589445680.0079348,
+                    589445683.0259652,
+                    589445686.0439956
+                ],
+                "Positions": [
+                    [
+                        -883.3647639832787,
+                        3235.9253823963995,
+                        -1734.2678063014955
+                    ],
+                    [
+                        -878.0820172062348,
+                        3241.143181714679,
+                        -1727.2955108387216
+                    ],
+                    [
+                        -872.7929026970128,
+                        3246.3375924365314,
+                        -1720.3106940936013
+                    ],
+                    [
+                        -867.4974605099152,
+                        3251.5085747718745,
+                        -1713.3134080300226
+                    ]
+                ],
+                "Velocities": [
+                    [
+                        1.7493370110442228,
+                        1.7327474299014805,
+                        2.308134335038546
+                    ],
+                    [
+                        1.7514526413807416,
+                        1.7250030655476496,
+                        2.312291077819496
+                    ],
+                    [
+                        1.7535556099591456,
+                        1.7172463879119024,
+                        2.316431067882768
+                    ],
+                    [
+                        1.7556459015231667,
+                        1.7094774543366804,
+                        2.3205542750988553
+                    ]
+                ]
+            },
+            "SunPosition": {
+                "SpkTableStartTime": 589445681.51695,
+                "SpkTableEndTime": 589445681.51695,
+                "SpkTableOriginalSize": 1,
+                "EphemerisTimes": [
+                    589445681.51695
+                ],
+                "Positions": [
+                    [
+                        -177834612.75017235,
+                        94014077.38389017,
+                        47921572.85876171
+                    ]
+                ],
+                "Velocities": [
+                    [
+                        -13.267042406345718,
+                        -20.96669045246765,
+                        -9.258799613448325
+                    ]
+                ]
+            }
+          }
+        }
+      }
+
+
+@pytest.fixture()
+def test_themisir_kernels():
+    kernels = get_image_kernels('I74199019RDR')
+    updated_kernels, binary_kernels = convert_kernels(kernels)
+    yield updated_kernels
+    for kern in binary_kernels:
+        os.remove(kern)
+
+@pytest.fixture()
+def test_themisvis_kernels():
+    kernels = get_image_kernels('V46475015EDR')
+    updated_kernels, binary_kernels = convert_kernels(kernels)
+    yield updated_kernels
+    for kern in binary_kernels:
+        os.remove(kern)
```

Comment:
You can make these just 1 test fixture, see voyager:

https://github.com/USGS-Astrogeology/ale/blob/master/tests/pytests/test_voyager_drivers.py#L288